requirements/requirements.txt

```diff
@@ -87,7 +87,7 @@ django-waffle==2.0.0 \
     --hash=sha256:1973801b2bfdebb2d4ac3a7e5871a51f4d4b91ef712bdf46cc65d89b9dc3c988 \
     --hash=sha256:e079cb530d3c8d62e204cfc08c7494a71cb08626630c09cd00e844bed1df0044
     # via -r requirements/requirements.in
-django==2.2.17 \
+django==2.2.18 \
     --hash=sha256:558cb27930defd9a6042133258caf797b2d1dee233959f537e3dc475cb49bd7c \
     --hash=sha256:cf5370a4d7765a9dd6d42a7b96b53c74f9446cd38209211304b210fe0404b861
```

Comment:
Snyk didn't update the hashes here.

Reply:
Yeah I'm trying to get it to stop opening auto PRs 😬